src/resources/language/_language-ja.yml

```diff
@@ -0,0 +1,88 @@
+toc-title-document: "目次"
+toc-title-website: "目次"
+
+section-title-abstract: "概要"
+section-title-appendices: "付録"
+section-title-footnotes: "脚注"
+section-title-references: "参考"
+section-title-reuse: "Reuse"
+section-title-citation: "引用"
+
+appendix-attribution-cite-as: "BibTeXで以下のように引用してください:"
+appendix-attribution-bibtex: "BibTeX"
+
+title-block-author-single: "作者"
+title-block-author-plural: "作者"
+title-block-affiliation-single: "所属"
+title-block-affiliation-plural: "所属"
+title-block-published: "公開"
+
+callout-tip-caption: "ヒント"
+callout-note-caption: "ノート"
+callout-warning-caption: "警告"
+callout-important-caption: "重要"
+callout-danger-caption: "危険"
+
+code-summary: "コード"
+
+code-tools-menu-caption: "コード"
+code-tools-show-all-code: "全てのコードを見せる"
+code-tools-hide-all-code: "全てのコードを隠す"
+code-tools-view-source: "ソースコードを見る"
+code-tools-source-code: "ソースコード"
+
+copy-button-tooltip: "コピー"
+
+repo-action-links-edit: "このページを編集"
+repo-action-links-source: "ソースコードを見る"
+repo-action-links-issue: "問題を報告する"
+
+search-no-results-text: "一致なし"
+search-matching-documents-text: "一致した文書"
+search-copy-link-title: "検索へのリンクをコピー"
+search-hide-matches-text: "追加の検索結果を非表示にする"
+search-more-match-text: "追加の検索結果"
+search-more-matches-text: "追加の検索結果"
+search-clear-button-title: "消去"
+search-detached-cancel-button-title: "取消"
+search-submit-button-title: "検索"
+
+crossref-fig-title: "図"
+crossref-tbl-title: "表"
+crossref-lst-title: "リスト"
```

Comment:
I could not think of an appropriate translation for "listing".
Is this used only for code blocks as follows?
https://quarto.org/docs/authoring/cross-references.html#code-listings

Reply:
Yes correct that's how its used

Reply:
Thank you for the explanation.
I think it would be easier to understand "code block" or something like that if we are only referring to code blocks, but why is it "Listing" in English?